services/frontend-v3/src/components/layouts/editProfileLayout/EditProfileLayoutView.jsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { useHistory } from "react-router-dom";
 import { PageHeader, Menu } from "antd";
 import { RightOutlined } from "@ant-design/icons";
-import { FormattedMessage } from "react-intl";
+import { FormattedMessage, injectIntl } from "react-intl";
```

Comment:
**Just a comment:**
I was wondering if it is bad to have both in the same file and found this:
[github issue comment](https://github.com/formatjs/formatjs/issues/942#issuecomment-345994000)

seems like its ok but let me know what you interpret from those comments

Reply:
I think its fine, but we could just use `FormattedMessage` and get the string like this, but not sure if it complicates the code

```jsx
<FormattedMessage id="language.code" defaultMessage="en">
	{ locale => (<PageHeader
			style={{
				padding: "0 0 15px 7px",
   				textTransform: "capitalize",
				textTransform: locale === "en" ? "capitalize" : "",
			}}
			title={<FormattedMessage id="edit.profile" />}
			/>)}
</FormattedMessage>
```

Reply:
I dont find that very clear tbh. so maybe we just stick to what we have